src/main/java/ai/rev/speechtotext/models/asynchronous/RevAiJobOptions.java

```diff
@@ -15,12 +17,22 @@ public class RevAiJobOptions {
 
   /** The media url where the file can be downloaded. */
   @SerializedName("media_url")
+  @Deprecated
```

Comment:
Is there notation for showing what customers should be using instead?

Reply:
If there is please point customers to the new parameter

Reply:
Yeah looks like we should be using @deprecated in the javadoc alongside this, so I added that as well